service/client.go

```diff
@@ -72,4 +72,14 @@ type Client interface {
 
 	// Returns whether we attached to a running process or not
 	AttachedToExistingProcess() bool
+
+	// Returns the address of a location expression
```

Comment:
Change to: "Returns concrete location information described by a location expression"